addons/hr_recruitment_integration_base/models/hr_job_post.py

```diff
@@ -0,0 +1,204 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from markupsafe import Markup
+
+from odoo import fields, models, _
+from odoo.exceptions import RedirectWarning
+
+
+class JobPost(models.Model):
+    _name = "hr.job.post"
+    _description = "Job Post"
+    _inherit = ['mail.thread', 'mail.activity.mixin']
+    _order = "write_date desc"
+
+    job_id = fields.Many2one(
+        'hr.job', string="Job", required=True, readonly=True)
+    recruiter_id = fields.Many2one(
+        'res.users', string="Recruiter", related='job_id.user_id', readonly=True)
+    platform = fields.Many2one(
+        'recruitment.platform', string="Platform",
+        required=True, readonly=True)
+    platform_icon = fields.Binary(related="platform.avatar_128")
+    apply_method = fields.Selection([
+        ('email', 'Email'),
+    ], string='Contact Method', required=True)
+    apply_vector = fields.Char(
+        string="Contact Point",
+        help="The email address, phone number, url to send applications to.")
+    post = fields.Html(string="Post", prefetch=False, required=True)
```

Comment:
Maybe something easier to grep.
```suggestion
    post_html = fields.Html(string="Post", prefetch=False, required=True)
```